ci/test-badge-server.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Clone and start the server
+git clone https://github.com/popperized/popper-badge-server.git /tmp/popper-badge-server
+pip install -r /tmp/popper-badge-server/requirements.txt
+python /tmp/popper-badge-server/app.py &
+
+# Use popper run to test the badge server
+source common-setup.sh
+init_test
+sed -i '$s/.*/badge-server-url: http:\/\/127.0.0.1:5000/' .popper.yml
+popper init mypipeone
+git config --global user.email "testuser@example.com"
+git config --global user.name "Test User"
+git remote add origin https://github.com/systemslab/popper.git
+git add .
+git commit -m "Add mypipeone"
+popper run
```

Comment:
can we check that we added a new status to the server (e.g. using `curl`)?

Reply:
There will be no need of that if we are using the badge in our readme